site_root/input/resources/examples/Task-se-001.json

```diff
@@ -0,0 +1,112 @@
+{
+  "resourceType": "Task",
+  "id": "se-001",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/kfdrc-sequencing-experiment"
+    ],
+    "versionId": "0.1.0"
+  },
+  "identifier": [
+    {
+      "system": "https://kf-api-dataservice.kidsfirstdrc.org/sequencing-experiments?genomic_file_id=GF_G3BJ1Y2J",
+      "value": "SRX1907265"
+    }
+  ],
+  "status": "completed",
+  "authoredOn": "2016-07-08T00:00:00+00:00",
+  "owner": {
+    "reference": "Organization/og-001"
+  },
+  "intent": "unknown",
```

Comment:
Maybe [`order`](https://www.hl7.org/fhir/codesystem-request-intent.html#request-intent-order), since a PI placed an order for this sequencing event?